contents/handbook/growth/marketing/posthog-style-guide.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Style guide
+sidebar: Handbook
+showTitle: true
+---
+
+This style guide explains our guidelines for contributions to PostHog's documentation, tutorials, and blog.
+
+## General principles
+
+### Assume almost nothing
+
+As you gain mastery of a product or feature, some things become second nature, but remember they weren't always so obvious. Call these out, and provide links to relevant docs or websites.
+
+Make it easy for your reader to implement their feature or solve their issue, whether they are an expert or just starting out with PostHog. 
+
+### "It's complicated" is not a useful observation
+
+When writing about complex topics, avoid simply explaining why it's complicated. It's frustrating for the reader and doesn't add value.
+
+### Avoid overly long intros
+
+Most articles can be improved by making the intro shorter and more direct. Two short paragraphs, or less, is preferable, though there will always be exceptions.
+
+## Brand guidelines
+
+### We're an analytics platform, not a product analytics tool
+
+Unless directly referring to our product analytics suite, substitute references to "product analytics" or "product analytics tool" with the phrase "analytics platform".
+
+> **Example:** “A data warehouse or product analytics tool like PostHog can be helpful here.” would become “A data warehouse or analytics platform like PostHog can be helpful here.
+
+Product OS is also an acceptable alternative – use your judgement based on the context.
+
+### Do not capitalize product features
+
+It's "product analytics" not "Product Analytics" and so on.
+
+## Style rules
+
+### Use American English
+
+PostHog is a global company. Our team and our customers are distributed around the world. For consistency, we use American English spelling and grammar.
+
+### Use the Oxford comma 
+
+Write "bananas, apples, and oranges", not "bananas, apples and oranges".
+
+Why does this matter? Consider the old joke:
+
+"There are two hard problems in computer science: naming things, cache invalidation, and off by one errors."
+
+That doesn't work without the Oxford.
```

Comment:
Not a big deal, but I don't get how this doesn't work without the Oxford? (A more obvious example is something like _When you come to the party can you bring Janet, a clown and an acrobat?_)